app.js

```diff
@@ -1,8 +1,23 @@
 const express = require("express");
-const { getTopics } = require('./controllers/controllers')
+const { getTopics, getArticlesByID } = require('./controllers/controllers')
 const app = express();
 app.use(express.json())
 
 app.get('/api/topics', getTopics)
 
+app.get('/api/articles/:article_id', getArticlesByID)
+
+
+app.use((err, req, res, next) => {
```

Comment:
This might begin to get a bit tangled if you carry on with conditional logic here. 
You could try separating out these concerns into separate middleware functions and invoking next ( ) to pass the error to the next middleware function if it doesn't satisfy your condition